app/editor/src/hooks/api-editor/editor/useApiHubConnection.ts

```diff
@@ -4,8 +4,10 @@ import React from 'react';
 export const useApiHubConnection = () => {
   const controller = React.useRef({
     getConnection: () => {
+      // TODO: here just for testing only for dev environment, and will be updated later on
+      const url = 'https://tno-dev.apps.silver.devops.gov.bc.ca/api/workOrderHub';
       const connection = new HubConnectionBuilder()
-        .withUrl(process.env.REACT_APP_API_URL + '/workOrderHub', { withCredentials: false })
+        .withUrl(url, { withCredentials: false }) // process.env.REACT_APP_API_URL + '/workOrderHub'
```

Comment:
This won't work.  I've overridden the configuration of the `HubPath` already in Openshift to `/api/workOrderHub` and you can test this theory by using your developer tools in a browser.  Just add a stop in the debugger and manually change the value.

`process = { env: { REACT_APP_API_URL: 'https://tno-dev.apps.silver.devops.gov.bc.ca/api' } };`

Reply:
Updated for testing on dev, thanks